app/react-components/Project.js

```diff
@@ -1,160 +1,271 @@
+        <DownloadButton
+          disabled={biospecimenCount === 0}
+          filename={biospecimenDataExportFileName}
+          dataExportExpands={biospecimenDataExportExpands}
+          url={`${authApi}/cases`}
+          activeText="Processing"
+          inactiveText={biospecimenCount === 0 ? 'No Biospecimen Data' : 'Download Biospecimen'}
+          fields={['case_id']}
+          filters={buildFilters(biospecimenDataExportFilters)}
         />
+      </Row>
 
-        <Column style={styles.column}>
+      <Row style={{ flexWrap: 'wrap' }}>
+        <span style={{ ...styles.column, ...styles.margin }}>
+          <EntityPageVerticalTable
+            className="summary-table"
+            style={styles.summary}
+            title="Summary"
+            thToTd={[
+              { th: 'Project ID', td: project.project_id },
+              { th: 'Project Name', td: project.name },
+              { th: 'Disease Type', td: project.disease_type },
+              { th: 'Primary Site', td: project.primary_site },
+              { th: 'Program', td: project.program.name },
+            ]}
+          />
+        </span>
+
+        <Column style={{ ...styles.column, ...styles.margin }}>
           <CountCard
             title="CASES"
-            count={$scope.project.summary.case_count}
-            icon={<CaseIcon style={{ width: '4rem', height: '4rem' }} />}
+            count={formatNumber(project.summary.case_count)}
+            icon={<CaseIcon style={styles.icon} />}
             style={styles.countCard}
+            onCountClick={() => {
+              window.location = `/search/c?filters=${
+                makeFilter([{ field: 'cases.project.project_id', value: project.project_id }])
+              }`;
+            }}
           />
 
           <CountCard
             title="FILES"
-            count={$scope.project.summary.file_count}
-            icon={<FileIcon style={{ width: '4rem', height: '4rem' }} />}
+            count={formatNumber(project.summary.file_count)}
+            icon={<FileIcon style={styles.icon} />}
             style={styles.countCard}
+            onCountClick={() => {
+              window.location = `/search/f?filters=${
+                makeFilter([{ field: 'cases.project.project_id', value: project.project_id }])
+              }`;
+            }}
           />
 
-          {$scope.project.annotation && <CountCard
+          <CountCard
             title="ANNOTATIONS"
-            count={$scope.project.annotations.pagination.total}
-            icon={<EditIcon style={{ width: '4rem', height: '4rem' }} />}
-            style={styles.countCard}
-          />}
+            count={formatNumber(project.annotations ? project.annotations.pagination.total : 0)}
+            icon={<EditIcon style={styles.icon} />}
+            style={{ ...styles.countCard, marginBottom: 0 }}
+            onCountClick={project.annotations && project.annotations.pagination.total > 0 && function() {
```

Comment:
This is throwing propType error because it's expecting a function, but getting `false` when no annotations. You could change it to pass a noop or conditionally add the prop

```
<CoundCard
   (...(project.annotations && project.annotations.pagination.total > 0) ? { onCountClick: ... } : {})
/>
```